etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/messages/SendMessageHelper.java

```diff
@@ -30,7 +37,14 @@ public void savePartnerMetadataForReceivedMessage(
             String stringMessageHash = String.valueOf(messageHash);
 
             partnerMetadataOrchestrator.updateMetadataForReceivedMessage(
-                    receivedSubmissionId, stringMessageHash, messageType);
+                    receivedSubmissionId,
```

Comment:
Same over here. Thoughts on passing an object with all the data?

Reply:
Same as above.